app/frontend/src/components/form/Step2.vue

```diff
@@ -739,3 +754,21 @@ export default {
 };
 </script>
 
+<style lang="scss" scoped>
+form {
+  h4:not(.heading-field-label) {
+    padding-bottom: 1em;
+  }
+  hr {
+    margin-bottom: 1.75em;
+    margin-top: .5em;
+  }
+  .row {
+    div[class^='col-'],
+    div[class*=' col-'] {
+      padding-bottom: 0;
+      padding-top: 0;
+    }
```

Comment:
good idea. looks better

Reply:
I think it has about the same effect as removing the error fields that you were mentioning? But wanted to do it on the public side as well (where I need the error fields). We can circle back and coordinate spacing with this change.